code/modules/mentor/verbs/mentorhelp.dm

```diff
@@ -0,0 +1,106 @@
+/client/verb/mentorhelp(msg as text)
+	set category = "Mentor"
+	set name = "Mentorhelp"
+
+	//clean the input msg
+	if(!msg)
+		return
+	msg = sanitize(copytext(msg,1,MAX_MESSAGE_LEN))
+	if(!msg || !mob)
+		return
+	if(src.prefs.muted & MUTE_MENTORHELP)
+		to_chat(src, "<font color='red'>You are unable to use mentorhelp (muted).</font>")
```

Comment:
Would `<span>` be better?

Reply:
Yeah, make that into `<span class='danger'>`